mono/mini/TestDriver.cs

```diff
@@ -51,6 +52,9 @@ public class TestDriver {
 				} else if (args [j] == "--exclude") {
 					exclude [args [j + 1]] = args [j + 1];
 					j += 2;
+				} else if (args [j] == "--exclude-test") {
+					exclude_test.Add (args [j + 1]);
```

Comment:
Why do we need another option, i.e. is there something the existing --exclude doesn't solve?